tests/components/homee/conftest.py

```diff
@@ -0,0 +1,64 @@
+"""Fixtures for Homee integration tests."""
+
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+from typing_extensions import Generator
+
+from homeassistant.components.homee.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+
+from tests.common import MockConfigEntry
+
+HOMEE_ID = "00055511EECC"
+HOMEE_IP = "192.168.1.11"
+TESTUSER = "testuser"
+TESTPASS = "testpass"
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        title=f"{HOMEE_ID} ({HOMEE_IP})",
+        domain=DOMAIN,
+        data={
+            CONF_HOST: HOMEE_IP,
+            CONF_USERNAME: TESTUSER,
+            CONF_PASSWORD: TESTPASS,
+        },
+        unique_id=HOMEE_ID,
+        version=1,
+        minor_version=1,
```

Comment:
```suggestion
```

Reply:
A question on the version, that  I have lying in my head for a while.
In the custom component, I will be at Version 3 by the time it moves to core.
Is it possible to start with 3 here, so when a user migrates from the custom comp, he can just keep the config_entry and does not have to reinstall?

Reply:
Nope, that's not possible

Reply:
Hmm, what will happen if someone just uninstalls the HACS repo without removing the config?

Reply:
Most likely a migration error.

In your custom component you could raise a repair if the version is above 2025.1, where you then just update the config entry to be shaped like we do, and automatically remove the custom component, which can lead to a batteries included migration for users